tests/e2e/utils/kubeflow_installation.py

```diff
@@ -291,31 +291,31 @@ def configure_kubeflow_pipelines(
         return
 
     cfg = load_yaml_file(file_path="./utils/rds-s3/metadata.yaml")
-    IAM_ROLE_ARN_FOR_IRSA = cfg["S3"]["roleArn"]
-
+    IAM_BACKEND_ROLE_ARN_FOR_IRSA = cfg["S3"]["backEndRoleArn"]
+    IAM_FRONTEND_ROLE_ARN_FOR_IRSA = cfg["S3"]["frontEndRoleArn"]
```

Comment:
nit: shall we rename the frontendRole to be generic - `profileRoleArn` or `userRoleArn` since Argo also uses it to upload artifacts?

optional: you can drop `IAM_` and `FOR_IRSA` from the names of these variables since it already has ROLE_ARN.